app-ios/tutanota/Sources/Utils/MurMurHash.swift

```diff
@@ -0,0 +1,304 @@
+//
+//  MurmurHash3.swift
+//  MurmurHash
+//
+//  Created by Daisuke T on 2019/02/06.
+//  Copyright © 2019 MurmurHash. All rights reserved.
```

Comment:
Do we need to [provide the license for `MurmurHash-Swift`](https://github.com/daisuke-t-jp/MurmurHash-Swift/blob/master/LICENSE) anywhere?

Reply:
As far as I know legal-wise, that covers it. :+1: